python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -229,152 +292,188 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
         if len(xs) == 0 or len(ys) == 0:
             return fig
 
-        # Points to use for the fit
-        fitPoints = np.where((xs > fitParams["xMin"]) & (xs < fitParams["xMax"])
-                             & (ys > fitParams["yMin"]) & (ys < fitParams["yMax"]))[0]
-
-        # Plot the initial fit box
-        ax.plot([fitParams["xMin"], fitParams["xMax"], fitParams["xMax"], fitParams["xMin"],
-                fitParams["xMin"]], [fitParams["yMin"], fitParams["yMin"], fitParams["yMax"],
-                fitParams["yMax"], fitParams["yMin"]], "k", alpha=0.3)
+        # Points used for the fit
+        fitPoints = fitParams["fitPoints"]
 
         # Add some useful information to the plot
-        bbox = dict(alpha=0.9, facecolor="white", edgecolor="none")
-        medMag = np.median(mags)
+        SNsUsedDict = {}
+        if hasattr(self.config.selectorActions, "catSnSelector"):
+            # Compute the effective S/N & mag cuts for the points used in the
+            # fit.
+            for SNBand in self.config.selectorActions.catSnSelector.bands:
+                SNsUsed = (catPlot[SNBand + "_" + plotInfo["SNFlux"]].values[fitPoints]
+                           / catPlot[SNBand + "_" + plotInfo["SNFlux"] + "Err"].values[fitPoints])
```

Comment:
When this gets ported to analysis_tools, there might be functions in place to do this.